platform/x11/detect.py

```diff
@@ -256,12 +256,8 @@ def configure(env):
         print("ALSA libraries not found, disabling driver")
 
     if env['pulseaudio']:
-        if (os.system("pkg-config --exists libpulse") == 0): # 0 means found
-            print("Enabling PulseAudio")
-            env.Append(CPPFLAGS=["-DPULSEAUDIO_ENABLED"])
-            env.ParseConfig('pkg-config --cflags --libs libpulse')
```

Comment:
We still need the `CFLAGS` to get the include path properly set up, otherwise it fails building:
```
In file included from platform/x11/os_x11.h:39:0,
                 from platform/x11/godot_x11.cpp:37:
./drivers/pulseaudio/audio_driver_pulseaudio.h:40:30: fatal error: pulse/pulseaudio.h: No such file or directory
 #include <pulse/pulseaudio.h>
                              ^
```